proto/executor.proto

```diff
@@ -78,3 +78,14 @@ message Limit {
 	// Limit the result to be returned.
 	optional uint64 limit = 1 [(gogoproto.nullable) = false];
 }
+
+message ExecutorExecutionSummary {
+	// Total time cost in this executor. Includes self time cost and children time cost.
+	optional uint64 total_time_ms = 1;
```

Comment:
In TiDB, the time unit it used is nanosecond seconds, I think this would be helpful in some fast cop tasks.

Reply:
The timer TiKV use will have a milliseconds resolution. But I changed it to ns now anyway.